python/GafferCyclesTest/IECoreCyclesPreviewTest/RendererTest.py

```diff
@@ -1429,5 +1433,112 @@ def testSurfaceAttributeWithGenericShaderType( self ) :
 
 		del plane
 
+	def __valueAtUV( self, image, uv, channelName = "" ) :
```

Comment:
The default value for `channelName` isn't useful here, since it will just cause a `KeyError` at L1444.

Reply:
Fixed.